### DISCUSSION TITLE
How do you use icon types with FontAwesome?

### DISCUSSION THREAD

**Discussion by michalvadak** [react-icons/react-icons#644] [Q&A]
FontAwesome has multiple types (https://fontawesome.com/icons/hotel?s=regular&f=classic). For example solid, regular, light. How do you use their types when importing `faHotel`?

**Comment by ProPhycient5** [comment]
Hi @michalvadak, can you try applying text CSS property to the icons component?

```
<FaBeer  style={{fontSize: "16px", fontWeight: "700"}}/>
```
